sed/binning.py

```diff
@@ -199,8 +199,8 @@ def bin_dataframe(
                         dfPartition,
                         None,
                         binAxes,
-                        binRanges,
                         nBins,
+                        binRanges,
                         hist_mode,
                     ),
                 )
```

Comment:
Is there any good reason to keep `legacy` and `lean` modes, apart from compatibility?

Reply:
They were there mostly to test which one performs best. For the time being I would leave them